src/metabase/util/i18n.clj

```diff
@@ -167,7 +167,7 @@
   (validate-number-of-args format-string args)
   `(SiteLocalizedString. ~format-string ~(vec args) {}))
 
-(def ^String ^{:arglists '([& args])} str*
+(def ^String ^{:arglists '([& args]) :redef true} str*
```

Comment:
We get `Caused by: java.lang.Exception: Premature i18n string lookup. Is there a top-level call to `trs` or `tru`?` error on startup if we direct-linking this function.

Reply:
That's concerning... any theories as to why that happened? I'd like to understand what's going on with this before we make a huge change like this

Reply:
I plan to look further into it next week. but for now, the error is from this logging: https://github.com/metabase/metabase/blob/f1fa503fa31657ac7009e44fdf4053ec3cf07442/src/metabase/util.cljc#L213

Reply:
That error is only supposed to be triggered during compilation: https://github.com/metabase/metabase/blob/f1fa503fa31657ac7009e44fdf4053ec3cf07442/src/metabase/util/i18n.clj#L172-L174

If it's happening outside of compilation, that seems like a problem

Also FWIW I don't think that message should be i18n'ed, it looks like we got around the error checking by disabling the message during compilation, but since the app DB isn't set up yet it will always just fall back to English